packages/sku/package.json

```diff
@@ -143,6 +143,7 @@
     "@vocab/react": "^1.0.1",
     "assert": "^2.0.0",
     "braid-design-system": "^31.0.0",
+    "memfs": "^3.5.1",
```

Comment:
I might have missed it, but where is this being used?

Reply:
Oh woops, I was playing around with it but didn't actually end up using it. Will remove.

Reply:
https://github.com/seek-oss/sku/pull/792/commits/c2a2779ab8ed647acab21873236dca67281e9cf1